.github/workflows/integration.yml

```diff
@@ -0,0 +1,35 @@
+name: Downstream Integration
```

Comment:
Totally an aesthetic thing, but the status section is super verbose about these, can we make them more like the existing GHA statuses?

Reply:
Any suggestions for a shorter name? Is "Downstream" too long too?

Reply:
(I agree that the current output is ugly but didn't know how to preserve clarity)

Reply:
I think "Downstream" is fine here

Reply:
Done

Reply:
Done